.github/copilot-instructions.md

```diff
@@ -0,0 +1,75 @@
+# GitHub Copilot Instructions
+
+This repository contains C# code.
+
+The purpose of the code is to provide a standalone SDK for interacting with Durable Functions.
```

Comment:
Not just Durable Functions! Durable Functions is an Azure Functions feature, and this SDK is used for both Azure Functions and non-Functions via DTS (and theoretically for other backends, like Dapr Workflow - we're not super opinionated about this since we just communicate with a gRPC contract).